nbviewer/frontpage.json

```diff
@@ -81,11 +86,6 @@
         "text": "Jaynes-Cummings model",
         "target": "github/jrjohansson/qutip-lectures/blob/master/Lecture-1-Jaynes-Cumming-model.ipynb",
         "img": "/static/img/example-nb/jaynes-cummings.png"
-      },
-      {
-        "text": "SymPy notebook",
-        "target": "github/ipython/ipython/blob/2.x/examples/Notebook/SymPy.ipynb",
-        "img": "/static/img/example-nb/sympy.png"
```

Comment:
This one probably didn't need to be cut as part of this PR, but this is referencing a soon to be old branch.